cli/src/main.rs

```diff
@@ -343,185 +343,211 @@ fn run() -> Result<(), CliError> {
-                                .required(true)
-                                .long("reject")
-                                .conflicts_with("accept")
-                                .help("Reject the proposal"),
-                        ),
-                )
+                .subcommand(SubCommand::with_name("list").about("List available templates"))
                 .subcommand(
-                    SubCommand::with_name("list")
-                        .about("List the circuits")
-                        .arg(
-                            Arg::with_name("url")
-                                .short("U")
-                                .long("url")
-                                .help("The URL of the Splinter daemon REST API")
-                                .takes_value(true),
-                        )
-                        .arg(
-                            Arg::with_name("member")
-                                .short("m")
-                                .long("member")
-                                .help("Filter the circuits by a node ID in the member list")
-                                .takes_value(true),
-                        )
-                        .arg(
-                            Arg::with_name("format")
-                                .short("f")
-                                .long("format")
-                                .help("Output format")
-                                .possible_values(&["human", "csv"])
-                                .default_value("human")
-                                .takes_value(true),
-                        ),
-                )
-                .subcommand(
-                    SubCommand::with_name("show")
-                        .about("Show a specific circuit or proposal")
-                        .arg(
-                            Arg::with_name("url")
-                                .short("U")
-                                .long("url")
-                                .help("The URL of the Splinter daemon REST API")
-                                .takes_value(true),
-                        )
-                        .arg(
-                            Arg::with_name("circuit")
-                                .help("The circuit ID of the circuit to be shown")
-                                .takes_value(true),
-                        )
-                        .arg(
-                            Arg::with_name("format")
-                                .short("f")
-                                .long("format")
-                                .help("Output format")
-                                .possible_values(&["human", "yaml", "json"])
-                                .default_value("human")
-                                .takes_value(true),
-                        ),
+                    SubCommand::with_name("show").about("Show a template").arg(
+                        Arg::with_name("name")
```

Comment:
Should add `.required(true)` for this arg (same for `arguments` subcommand).

Reply:
addressed